index.html

```diff
@@ -27,25 +27,44 @@
         font-family: Arial, Helvetica, sans-serif;
         font-size: 1.2rem;
         line-height: 1.6;
+        
       }
       /*
         These styles make sure that any super wide images
         don't overflow the page width
       */
       img {
-        width: 100%;
+        width: 50%;
         display: block;
         margin: 0 auto;
+
       }
+
+
+      table {
+            width: 100%;
+            border-collapse: collapse;
+            margin: 5px 0;
+        }
+
+        th, td {
+           
+            padding: 12px;
+            text-align: left;
+            
+        }
     </style>
   </head>
   <body>
+ 
     <!--
 
       LIST ANY CITATIONS HERE
 
-        * A
-        * B
+        * all the word texts are from chatGPT minus the table and the footer
+        * images are from 
+        * Apple Store Online. Apple (CA). (n.d.). 
+        https://www.apple.com/ca/store?afid=p238%7CsEnMtlDRF-dc_mtid_1870765e38482_pcrid_675586750318_pgrid_39429562932_pntwk_g_pchan__pexid__&amp;cid=aos-ca-kwGO-brand--slid---product- 
```

Comment:
Not a huge deal - but when citing ChatGPT you should include the prompts that you used to generate whatever content you included.